packages/lib/src/core/Errors/SRPanel.tsx

```diff
@@ -19,7 +19,7 @@ export class SRPanel extends BaseElement<SRPanelProps> {
         showPanel: false,
         id: 'ariaLiveSRPanel',
         ariaAttributes: {
-            'aria-relevant': 'all',
+            'aria-relevant': 'additions',
```

Comment:
Can you confirm that all the errors still get read out when the (Card) PM is completely empty and the "Pay" button is pressed?
(I thought that was why I'd used "all" instead of "additions" originally - but it was a long time ago...!)

Reply:
I have just tested the Pay button scenario. It does not change with `aria-relevant`.
However It's not always reading from aria-live. I will double check exactly if is a new behaviour.